include/SDL3/SDL_gpu.h

```diff
@@ -2807,15 +2808,17 @@ extern SDL_DECLSPEC void SDLCALL SDL_EndGPURenderPass(
  * Begins a compute pass on a command buffer.
  *
  * A compute pass is defined by a set of texture subresources and buffers that
- * will be written to by compute pipelines. These textures and buffers must
- * have been created with the COMPUTE_STORAGE_WRITE bit. All operations
+ * may be written to by compute pipelines. These textures and buffers must
+ * have been created with the COMPUTE_STORAGE_WRITE bit or the COMPUTE_STORAGE_SIMULTANEOUS_READ_WRITE bit. All operations
  * related to compute pipelines must take place inside of a compute pass. You
  * must not begin another compute pass, or a render pass or copy pass before
  * ending the compute pass.
  *
- * A VERY IMPORTANT NOTE Textures and buffers bound as write-only MUST NOT be
- * read from during the compute pass. Doing so will result in undefined
- * behavior. If your compute work requires reading the output from a previous
+ * A VERY IMPORTANT NOTE - Reads and writes in compute shaders are NOT implicitly synchronized.
```

Comment:
This sounds like the app shouldn't use this feature. When is it safe to do so? Where can I find out more?

Reply:
Not very much, but I feel like many tutorials and books could definitely be built around this API. :)

Reply:
Me and Caleb are planning a tutorial series, I would honestly love to write a book too. I'll try to add a few clarifications for now.